docker-compose.yml

```diff
@@ -1,5 +1,10 @@
 version: '3'
 services:
+  api:
```

Comment:
let's use `mock-api` or smth

Reply:
We need to share the name of the mock & not mocked api so github can connect to both. The .env variable for BACKEND_ADDRESS is set to =api, and we can't really change that during run time?

Reply:
Excellent point! I forgot about the docker-based networking. Fine to leave as is for now.